client/signup/steps/woocommerce-install/index.tsx

```diff
@@ -11,4 +11,7 @@ export interface WooCommerceInstallProps {
 	queryObject: {
 		siteSlug: string;
 	};
+	signupDependencies: {
+		siteSlug: string;
+	};
```

Comment:
This isn't being used but wanted to give us some prior art for any other steps coming up in the next cycle with onboarding flows that will need to take advantage of how this works.